images/macos/scripts/build/install-common-utils.sh

```diff
@@ -17,7 +17,12 @@ for package in $common_packages; do
         xcbeautify_path=$(download_with_retry "https://raw.githubusercontent.com/Homebrew/homebrew-core/d3653e83f9c029a3fddb828ac804b07ac32f7b3b/Formula/x/xcbeautify.rb")
         brew install "$xcbeautify_path"
     else
-        brew_smart_install "$package"
+        if [[ $package == "packer" ]]; then
```

Comment:
Can this logic be folded into `brew_smart_install`? I assume that internally it invokes `brew install` for the provided package so if we specify `hashicorp/tap/packer` in the template then it would install the thing. Perhaps some changes might be needed in the `brew_smart_install` to handle compound name with `/`.

Reply:
> Can this logic be folded into brew_smart_install?

In general, yes, but it doesn't make much sense - the complication will be quite large given that this is the corner case and will exhaust itself soon. Now for `packer` there is both a formula and a tap, which makes the situation quite funny. It's easier to hardcode until the formula is deprecated completely - after that it will be possible to simplify the installation and remove the exception.
Later it will be necessary to make a separate work item to improve the function for working with taps. But now we need to quickly unlock the generation. 😞

Reply:
> Later it will be necessary to make a separate work item to improve the function for working with tap

@erik-bershel How do we track this work? Is there an issue for it?

Reply:
> How do we track this work? Is there an issue for it?

I placed an issue on the internal board to check on it once formulae deprecated.